_posts/2023-02-15-#26441.md

```diff
@@ -10,15 +10,47 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- A Bitcoin peer may have connections with other peers. By default (in Bitcoin Core), it allows up to 125 connections, 
+  8 of which are outbound.
+
+- There are different kind of connections. E.g: block-relay-only, inbound, outbound-full-relay, manual, feeler, 
+  block-relay and addr-fetch.
+
+- You can use RPC `getpeerinfo` to get data about each connected network peer and check the connection type, permissions
+  and other informations.
+
+- `-whitelist` flag allows to add permission flags to the peers connecting from the given IP address or CIDR-notated
```

Comment:
whitelisting and permissions are sometimes used interchangeably. Perhaps it's useful to expand a bit on this in the notes?

Reply:
you mean `-whitelist` and `-whitebind`?